conf/machine/nitrogen8m.conf

```diff
@@ -31,8 +31,14 @@ SPL_BINARY = "spl/u-boot-spl.bin"
 UBOOT_DTB_NAME ?= "imx8mq-nitrogen8m.dtb"
 UBOOT_MAKE_TARGET = ""
 UBOOT_SUFFIX = "bin"
+
+#Used to define the uboot defconfig for different board variants.
+#The naming is here as a safeguard mechanism so that a user doesn't build and flash the wrong configuration.
+#This method will be optimized in the future, but for now, use this variable to define the defconfig to be built.
+BOUNDARY_DEVICES_UBOOT_DEFCONFIG ?= "nitrogen8m"
```

Comment:
```suggestion
# Used to define the U-Boot defconfig for different board variants. The naming
# is here as a safeguard mechanism so that a user doesn't build and flash the
# wrong configuration.
#
# This method will be optimized in the future, but for now, use this variable to
# define the defconfig to be built.
BOUNDARY_DEVICES_UBOOT_DEFCONFIG ?= "nitrogen8m"
```